app/hotels/package.json

```diff
@@ -7,11 +7,14 @@
     "@kiwicom/react-native-app-config": "^0",
     "@kiwicom/react-native-app-redux": "^0",
     "@kiwicom/react-native-app-shared": "^0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
     "geolib": "2.0.24",
     "idx": "2.2.0",
     "lodash": "^4.17.4",
     "moment": "^2.20.1",
     "querystring": "^0.2.0",
+    "react-dom": "^16.2.0",
```

Comment:
3 questions... 😊 

1. Why do we need it? We do not work with DOM as you know it. Sounds odd.
2. Why 16.2? We are using React 16.0. (we have the same error with `react-test-renderer`)

Reply:
1. Enzyme needs it
2. I could probably downgrade it
3. I will move it to root

Reply:
> Enzyme needs it